src/vs/workbench/contrib/chat/common/chatViewModel.ts

```diff
@@ -59,6 +59,7 @@ export interface IChatRequestViewModel {
 export interface IChatResponseMarkdownRenderData {
 	renderedWordCount: number;
 	lastRenderTime: number;
+	isPlaceholder: boolean;
```

Comment:
This feels a little wrong... this interface is metadata that comes from the rendering process and drives the progressive render, not the actual data to be rendered, which it feels like `isPlaceholder` would be part of